CharlieBackend.AdminPanel/Views/Students/AllStudents.cshtml

```diff
@@ -29,56 +31,106 @@
+                                                            <i class="fas fa-pen"></i>
+                                                            <span>Edit </span>
                                                         </a>
-                                                    }
+
+                                                        @if (@item.IsActive)
+                                                        {
+
+                                                            <a class="dropdown-item text-dark" id="disableStudent" data-toggle="modal" data-target="#disableStudentModalWindow" data-item-id="@item.Id">
+                                                                <i class="fas fa-trash"></i>
+                                                                <span>Disable </span>
+                                                            </a>
+                                                        }
+                                                    </div>
                                                 </div>
-                                            </div>
-                                        </td>
-                                    </tr>
+                                            </td>
+                                        </tr>
+                                    }
+                                    else
+                                    {
+                                        <tr class="row d-none" id="inactiveStudentsRow">
+                                            <td class="col-4 text-truncate">@item.Email</td>
+                                            <td class="col-3 text-truncate">@item.FirstName</td>
+                                            <td class="col-3 text-truncate">@item.LastName</td>
+                                            <td class="col">
+                                                <div class="dropdown">
+                                                    <button class="btn btn-outline-secondar dropdown-toggle"
+                                                            type="button" id="dropdownMenuButton" data-toggle="dropdown"
+                                                            aria-haspopup="true" aria-expanded="false">
+                                                        <i class="fas fa-cog"></i>
+                                                    </button>
+                                                    <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+
+                                                        <a class=" dropdown-item" asp-action="UpdateStudent" asp-controller="Students" asp-route-id="@item.Id">
+                                                            <i class="fas fa-pen"></i>
+                                                            <span>Edit </span>
+                                                        </a>
+
+                                                        @if (@item.IsActive)
```

Comment:
there won`t be any active students, because in else block we would have only inactive ones